textile/features.textile

```diff
@@ -78,6 +78,7 @@ h2(#endpoint-configuration). Client library endpoint configuration
 *** @(REC1b2)@ If the @endpoint@ option is a domain name, determined by it containing at least one period (@.@) then the @primary domain@ is the value of the @endpoint@ option.
 *** @(REC1b3)@ Otherwise, if the @endpoint@ option is a non-production routing policy name of the form @nonprod:[name]@ then the @primary domain@ is @[name].realtime.ably-nonprod.net@.
 *** @(REC1b4)@ Otherwise, if the @endpoint@ option is a production routing policy name of the form @[name]@ then the @primary domain@ is @[name].realtime.ably.net@.
+**** @(REC1b4a)@ Unless the @endpoint@ option is a routing policy name of @sandbox@ then the @primary domain@ is @sandbox.realtime.ably-nonprod.net@ to maintain backward compatibility.
```

Comment:
This is kindof ambiguously phrased -- I'd usually parse "Unless X then Y" as "Y applies if !X". which here would mean, set the primary domain to sandbox.realtime.ably-nonprod.net unless endpoint="sandbox". (sure, with common sense it's clear it's not that, but not good practice)

I'd be tempted to put this as an REC1b5 between b3 and b4 at the same level, so it's part of the one if-else waterfall, saying "Otherwise, if the `endpoint` option is `sandbox` then the `primary domain` is `sandbox.realtime.ably.net`"? Or maybe modify REC1b3 to say "...a non-production routing policy name of the form `nonprod:[name]`, or 'sandbox', ...". wdyt?

Reply:
Agreed, we should simplify phrasing the statement

Reply:
Agree with this and have updated.